### DISCUSSION TITLE
Unclear expo-fonts docs

### DISCUSSION THREAD

**Discussion by dominictobias-bullish** [expo/expo#28670] [Proposals and Ideas]
Hi, your readme here: https://docs.expo.dev/versions/latest/sdk/font/

Is unclear on the relationship between declaring in app.json and `useFonts`. Do we always have to declare in both or is it one or the other? What's the difference in behavior?

Thanks

**Comment by raajnadar** [comment]
You can go through the docs - https://docs.expo.dev/develop/user-interface/fonts/#use-a-custom-font

They are 2 different approaches.

**Reply by dominictobias-bullish** [reply]
Thanks that explains it!
